notebooks/contrib-dev/Stochastic_Simu_Chem_Rxn.ipynb

```diff
@@ -0,0 +1,2654 @@
nnnqK0NBQvv/+e7Zt28bs2bPx8vqngPfmm2/mlltu4f3330ev15tXPP7666+5++676dmzZ1X/GK4oTIarhKgTlFIk5yZzOvN0iUdiViLnss+RnJtcoQRGQ8PX1Rc/Vz8C3ALwd/MnwPXSV7cA/F398XfzNycwLg5VO+QuRFXQlFI1vnS8b9++/PXXX2We69OnD+vXry9xbMuWLbzyyivExcWhaRrdu3fn7bffpkOHDqVen5SUxAsvvMCqVavIysqiVatWPP/889x+++2l2ubl5fHmm28yf/58zp49S2hoKPfeey8vvPACTk4V70LNyMjAx8cHvV6Pt7d3hV93JTkFRbR67XcA9rw+CB836dIVorZSSnE+5zwn9Cc4lXGqRDKTlJV01XVHHDVHgtyDCPYIpr57feNXj/oEuwcT7BFMoHsgfq5+OOpqxf9zhSihMp+htSLJsTdVkeQAdHpzNclZBSyf3JM2oT5Wu64QomoYlIGkrCROpJ/ghP4Ex9OPc0JvfJ5dmF3u63SajmD3YMK8wmjo1dD8tYFHA+p71Mff1R8HnUM1vhMhqk9lPkMljbcjDX3dSc4qIDEtR5IcIWqY3KJcjqUd41DaIQ6nHuZg6kGOph0tt1fGQXMgzCuMCO8IcyJjeoR6hkoBrhAVIEmOHQnzc2f36XSZYSWEjeUV5XEw9SB7L+7lQMoBDqce5mTGyTJrZZx0TjT2aUykTySR9SJp4tOESJ9IGnk3kkRGCAtJkmNHzNPIZYaVENVGKUViZiJ7kvew9+Je9l7cy+HUwxSp0ovX+bn60cKvBc39mtPCtwUt/FoQ7h0utTFCVBH5m2VHZIaVEFXPoAwcTTvKtnPb2HpuK7sv7CYtP61UO39Xf9oFtqNNQBta+BkTmkC3QJlOLUQ1kiTHjoT5mtbKkeEqIaxFKcXx9ONsPbeVbee2sf38dtLz00u0cdI50dK/Je0C2nFd4HW0C2xHiEeIJDRC2JgkOXYkzM84XJWYloNSSv6BFeIa6fP1bEraxIbEDcSejSU1L7XEeTdHNzrU70CX4C50rN+Rln4tcXZwtlG0QojySJJjRxrUc0OnQV6hgYtZ+QR5udo6JCFqBaUUR9KOsDFpIxsSN7Dn4p4SRcKuDq60D2pPl+AudA7uTOuA1jjppChYiJpOkhw74uSgI8THjaT0XE6n5kqSI8QVFBYXEncujj9P/cnGpI2cyz5X4nxT36b0Du1Nz9CetAtsJz01QtRCkuTYmYa+xiQnMS2Hjo18bR2OEDVKYXEhW85u4Y/4P1h3eh0ZBRnmc64OrnQJ6UKfhn3oFdqLEM8QG0YqhLAGSXLsTJifO3EnU2WGlRCXGJSBbee2sfzEctaeWktmQab5nL+rPzeE30CfsD50Ce6Cq6P0fgphTyTJsTPmGVayIKCo446nH2fZ8WWsOLmixFBUgFsAA8IHMChiEB2COsj2B0LYMUly7Ey4vywIKOqu7MJsVpxYweKji9mfst983MvJixsb38iQxkMksRGiDpEkx86YenJOyXCVqEMOphxk0ZFFrDixgpwi4+++o+ZIz9CeDG0ylD5hfXBxcLFxlEKI6iZJjp0J9zcmOWfScykoMuDsqLNxREJUjcLiQn6L/42FBxeyL2Wf+XiEdwSjmo1iaJOh+Ln62TBCIYStSZJjZwI9XXBzciC3sJgz6blEBHjYOiQhrCo9L50fj/zID4d+4GLuRQAcdY4MDB/I6Oaj6VS/kyyEKYQAJMmxO5qmEe7nzuHzmSSk5kiSI+xGvD6e7w58x9LjS8krzgMgyC2IO1veyciokfi7+ds4QiFETSNJjh0K9zcmOadSsoFAW4cjhEWOpR1j1t5Z/Bb/GwoFQEu/ltzT6h4GRwzGyUFWHhZClE2SHDsUfmk38oQUKT4Wtdeh1EPM2juL1Qmrzcf6NOzD+NbjZUhKCFEhkuTYoUb+MsNK1F5H0o4wc9dM1p9ebz42sNFAHmz3IC38WtgsLiFE7SNJjh0y9eRIkiNqk7NZZ/lk9ycsO74MhUKn6bgx4kYebPsgUb5Rtg5PCFELSZJjhxr5G4uNT6XmoJSSbn1Ro6XnpTP779l8f+h7CgwFAAxqNIjHrn+Mxj6NbRydEKI2kyTHDoXWc0OnQU5BMRez8mU3clEjFRoKWXhwIV/u+ZLMQuN+Up2DO/NkhydpG9jWxtEJIeyBJDl2yNlRR4iPcTfy06k5kuSIGif2bCxvx73NCf0JAJr5NuPJjk/So0EP6XkUQliNJDl2qpG/O0npuSSk5NCxkaz6KmqGs1lnmbF9Bn8k/AGAr4svT3R4ghFRI2Q/KSGE1UmSY6fC/dyJOZ4i08hFjVBkKGLe/nl8ufdLcoty0Wk6bm9+O4+2fxQfFx9bhyeEsFOS5Ngp0x5Wp2WGlbCxw6mHeXXzqxxMPQhAh6AOvNT1JZr7NbdxZEIIeydJjp1q5GecYZUgSY6wkYLiAr7c+yXf/P0NRaoIb2dvnu38LMObDJe6GyFEtZAkx07JqsfClvZe3Mtrm1/juP44AAPCB/By9MsEuAXYODIhRF0iSY6dMg1XJWflk51fhIeL/FGLqldkKOKrvV/xxd4vMCgDfq5+vNz1ZQZFDLJ1aEKIOkg++eyUj5sT9dydSM8p5HRaDi2CvW0dkrBzZ7LO8MLGF9h1YRcANzW+iZe6vEQ913q2DUwIUWdJkmPHwv3cSc/Rk5AiSY6oWqtOruKNLW+QWZiJh5MHr0S/wi2Rt9g6LCFEHSdJjh0L93Nnb6KeU1KXI6pITmEOb8W9xa/HfwWgXWA73un1DmFeYTaOTAghJMmxaxGX9rCKT8m2cSTCHiVkJDBl3RSOpR9Dp+mY2HYiD1/3MI46+WdFCFEzyL9GdiwiQJIcUTX+Ov0XL258kczCTALcAni397t0Cu5k67CEEKIESXLsWMSlGVbxyTJcJazDoAx8secLPt/zOQDtA9vzXt/3CHIPsnFkQghRmiQ5dszUk3NGn0teYTGuTrI
```

Comment:
This would be much more effective if you made a plot that compared the solution profiles. You could make subplots for species A, B, and C. Then, on each subplot, you can compare the forward Euler, Crank-Nicolson, and a higher-order method (with scipy).